strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/storage/repository/Repository.java

```diff
@@ -359,4 +344,75 @@ public void setHttpConnectionPool(HttpConnectionPool httpConnectionPool)
     {
         this.httpConnectionPool = httpConnectionPool;
     }
+
+    public String getDetachAll()
+    {
+        return detachAll;
+    }
+
+    public void setDetachAll(String detachAll)
+    {
+        this.detachAll = detachAll;
+    }
+
+    public Long getVersion()
+    {
+        return version;
+    }
+
+    public void setVersion(Long version)
+    {
+        this.version = version;
+    }
+
+    public boolean isAllowsForceDeletion()
```

Comment:
@AlexOreshkevich,

Could you please restore the original method names here (and below)? 

* `isAllowsForceDeletion()` --> `allowsForceDeletion()`
* `isAllowsRedeployment()` --> `allowsRedeployment()`
* etc, etc....